hazelcast/proxy/topic.py

```diff
@@ -70,6 +72,27 @@ def publish(self, message: MessageType) -> Future[None]:
         request = topic_publish_codec.encode_request(self.name, message_data)
         return self._invoke(request)
 
+    def publish_all(self, messages: typing.Sequence[MessageType]) -> Future[None]:
+        """Publishes the messages to all subscribers of this topic.
+
+        Args:
+            messages: The messages to be published.
+        """
+        try:
+            if messages is None:
+                raise NullPointerError("Null message is not allowed!")
+            data_list = []
+            for m in messages:
+                data = self._to_data(m)
+                if data is None:
+                    raise NullPointerError("Null message is not allowed!")
```

Comment:
same as above, we can use `check_not_none`. Also, we need to check for none values before calling `to_data`

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/pull/570/commits/945b96544c5d77cf52013b86047f8bb3cf29eb3e.